### ISSUE TITLE
[BUG] BackgroundJobServer stopped while trying to insert JOBRUNR_METADATA

### ISSUE THREAD

**Issue by keldkemp** [jobrunr/jobrunr#347]
**Are you running the latest version of JobRunr?**
Yes

**Describe the bug**
```
2022-02-01 07:02:10.991 ERROR 7 --- [ool-37-thread-1] org.jobrunr.server.ServerZooKeeper       : An unrecoverable error occurred. Shutting server down...

org.jobrunr.storage.StorageException: java.sql.SQLException: ORA-12899: value too large for column "DEV"."JOBRUNR_METADATA"."NAME" (actual: 37, maximum: 36)

	at org.jobrunr.storage.sql.common.DefaultSqlStorageProvider.saveMetadata(DefaultSqlStorageProvider.java:144)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:177)
	at java.base/java.util.HashMap$KeySpliterator.forEachRemaining(HashMap.java:1694)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:497)
	at org.jobrunr.server.dashboard.DashboardNotificationManager.notify(DashboardNotificationManager.java:40)
	at org.jobrunr.server.ServerZooKeeper.lambda$signalBackgroundJobServerAlive$0(ServerZooKeeper.java:94)
	at java.base/java.util.Optional.ifPresent(Optional.java:176)
	at org.jobrunr.server.ServerZooKeeper.signalBackgroundJobServerAlive(ServerZooKeeper.java:94)
	at org.jobrunr.server.ServerZooKeeper.signalBackgroundJobServerAliveAndDoZooKeeping(ServerZooKeeper.java:76)
	at org.jobrunr.server.ServerZooKeeper.run(ServerZooKeeper.java:49)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:305)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:832)
Caused by: java.sql.SQLException: ORA-12899: value too large for column "DEV"."JOBRUNR_METADATA"."NAME" (actual: 37, maximum: 36)

	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:509)
	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:461)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1104)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:553)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:269)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:655)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:270)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:91)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:970)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1205)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3666)
	at oracle.jdbc.driver.T4CPreparedStatement.executeInternal(T4CPreparedStatement.java:1426)
	at oracle.jdbc.driver.OraclePreparedStatement.executeLargeUpdate(OraclePreparedStatement.java:3756)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:3736)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:1063)
	at com.zaxxer.hikari.pool.ProxyPreparedStatement.executeUpdate(ProxyPreparedStatement.java:61)
	at com.zaxxer.hikari.pool.HikariProxyPreparedStatement.executeUpdate(HikariProxyPreparedStatement.java)
	at jdk.internal.reflect.GeneratedMethodAccessor1263.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at pro.greendata.core.jdbc.logging.SimpleStatementProxyLogic.performQueryExecutionListener(SimpleStatementProxyLogic.java:345)
	at pro.greendata.core.jdbc.logging.SimpleStatementProxyLogic$1.execute(SimpleStatementProxyLogic.java:152)
	at net.ttddyy.dsproxy.listener.MethodExecutionListenerUtils.invoke(MethodExecutionListenerUtils.java:41)
	at pro.greendata.core.jdbc.logging.SimpleStatementProxyLogic.invoke(SimpleStatementProxyLogic.java:149)
	at pro.greendata.core.jdbc.logging.SimplePreparedStatementInvocationHandler.invoke(SimplePreparedStatementInvocationHandler.java:41)
	at com.sun.proxy.$Proxy164.executeUpdate(Unknown Source)
	at org.jobrunr.storage.sql.common.db.Sql.insertOrUpdate(Sql.java:138)
	at org.jobrunr.storage.sql.common.db.Sql.insert(Sql.java:111)
	at org.jobrunr.storage.sql.common.MetadataTable.save(MetadataTable.java:40)
	at org.jobrunr.storage.sql.common.DefaultSqlStorageProvider.saveMetadata(DefaultSqlStorageProvider.java:140)
	... 22 common frames omitted
Caused by: oracle.jdbc.OracleDatabaseException: ORA-12899: value too large for column "DEV"."JOBRUNR_METADATA"."NAME" (actual: 37, maximum: 36)

	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:513)
	... 51 common frames omitted

2022-02-01 07:02:11.014  INFO 7 --- [      Thread-13] org.jobrunr.server.BackgroundJobServer   : BackgroundJobServer and BackgroundJobPerformers stopped

```

**Environment**
I'm using JobRunr version: 4.0.7
I'm running on JRE / JDK (e.g. OpenJDK 1.8.0_292): OpenJDK 14.0.2
I'm using the following StorageProvider (e.g. Oracle / MongoDB / ...): Oracle

**How to Reproduce**
Until I realized. Trying to figure out what he's trying to insert. Maybe you can give a hint?

**Expected behavior**
The insert will succeed. The BackgroundJobServer will continue.

**Comment by rdehuyss** [comment]
Confirmed bug. I don't know how that one sneaked in to be honest. Will be fixed in JobRunr 4.0.8. 

I assume you are running on shared infrastructure (either on-premise or in the cloud)? JobRunr detected a Cpu Allocation Irregularity and wanted to show that in the dashboard but that went wrong.

> JobRunr detected some CPU Allocation Irregularities (e.g. due to a very long garbage collect cycle or due to CPU starvation on shared hosting). This may result in unstable behaviour of your JobRunr cluster.

If you have ran the SQL scripts manually, you can already fix it with the new SQL script (see below).

```
ALTER TABLE jobrunr_metadata
    MODIFY name  NVARCHAR2(92);
```

**Comment by keldkemp** [comment]
Hi! 
Yes, we use Kubernetes. 

Thanks!

**Comment by rdehuyss** [comment]
No worries, everything works out now? Did you apply the SQL script?

Feel free to star the repo if you're enjoying JobRunr ;-)

**Comment by keldkemp** [comment]
@rdehuyss, We will wait for version 4.0.8. After the update, I will observe and unsubscribe about the results

**Comment by rdehuyss** [comment]
fyi - I've released 4.0.8.
